config/routes.rb

```diff
@@ -1,8 +1,10 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
-  get "static_page/home"
-  get "static_page/help"
-  get "static_page/about"
+  get "users/new"
   root "static_page#home"
+  get  "/help",    to: "static_page#help"
+  get  "/about",   to: "static_page#about"
+  get  "/contact", to: "static_page#contact"
+  get  "/signup",  to: "users#new"
```

Comment:
`,    to`...
1 space thôi em
với đổi line khai báo `root` lên trên `get..` nhé